src/api/app/models/event/request_delete.rb

```diff
@@ -2,6 +2,10 @@ module Event
   class RequestDelete < Request
     self.message_bus_routing_key = 'request.delete'
     self.description = 'Request deleted (admin only)'
+
+    def subject
+      "Request #{payload['number']} deleted by #{payload['who']} (#{actions_summary})"
```

Comment:
```suggestion
      "Request #{payload['number']} deleted by #{payload['who']}"
```
I'm not really sure if we should give many details about the actions of a request. I would prefer to show the details in the body of an email instead of in the subject.